go.mod

```diff
@@ -229,6 +229,9 @@ replace (
 replace (
 	// use cosmos fork of keyring
 	github.com/99designs/keyring => github.com/cosmos/keyring v1.2.0
+
+	// release/chain-main/v0.50.x
+	github.com/cosmos/cosmos-sdk => github.com/crypto-org-chain/cosmos-sdk v0.50.6-0.20250515092613-f67eec43305b
```

Comment:
Why are we using a different v0.50.x branches (release/v0.50.x and release/chain-main/v0.50.x) for Cronos EVM and Cronos PoS chain

Reply:
Should we migrate it later to use the same SDK branch for simplicity and less maintenance?